Resources/doc/cookbook/enable_multi_upload.rst

```diff
@@ -0,0 +1,44 @@
+Enable Multi Upload
+===================
+
+The Multi Upload provides a way to create multiple media elements at once.
+
+.. figure:: ../images/sonata_multi_upload.png
+   :align: center
+   :alt: The Sonata Multi Upload
+
+To enable this feature your provider must implement the ``MultiUploadInterface`` and the provider template ``multi_upload_input`` needs to be set

```

Comment:
- Please add a final dot.
- [Each line should break approximately after the first word that crosses the 72nd character (so most lines end up being 72-78 characters);](https://symfony.com/doc/current/contributing/documentation/standards.html#main)